tests/from_msa/test_cluster_sequences.py

```diff
@@ -440,33 +433,9 @@ def test_GivenSequencesWithSameKmerCounts_ClusteringInterrupted(self):
         assert not sequences_are_one_reference_like(sequences)
 
         alignment = make_alignment(sequences)
-        result = kmeans_cluster_seqs_in_interval([0, len(sequences[0])], alignment, 7)
-        self.assertTrue(result.no_clustering)
-
-
-class TestMergeSequences(TestCase):
-    def setUp(self):
-        self.long_seqs = ["AATAA", "TTAAA"]
-        self.short_seqs = ["AA", "TT"]
-        self.first_seq = self.long_seqs[0]
-
-    def test_GivenFirstSeqNotInList_Fails(self):
-        with self.assertRaises(ValueError):
-            merge_sequences(self.short_seqs, self.long_seqs, first_seq="TTTTT")
-
-    def test_GivenShortFirst_FirstSeqIsFirstSeqOut(self):
-        result = merge_sequences(
-            self.short_seqs, self.long_seqs, first_seq=self.first_seq
-        )
-        expected = ["AATAA", "AA", "TT", "TTAAA"]
-        self.assertEqual(result.sequences, expected)
-
-    def test_GivenLongFirst_FirstSeqIsFirstSeqOut(self):
-        result = merge_sequences(
-            self.long_seqs, self.short_seqs, first_seq=self.first_seq
-        )
-        expected = ["AATAA", "TTAAA", "AA", "TT"]
-        self.assertEqual(result.sequences, expected)
```

Comment:
These tests were there for a reason!!
The first input sequence in the alignment needs to be the first path in the output prg. gramtools relies on this as the 'reference genome' (carries over the input reference genome sequence in the alignment). Why are these removed??

Reply:
Now added back in the last commits. Please recheck